docs/source/traitsui_user_manual/examples/tree_editor.py

```diff
@@ -0,0 +1,184 @@
+# (C) Copyright 2004-2021 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+# tree_editor.py -- Example of a tree editor
```

Comment:
maybe this should end up in `traitsui/examples/demo/StandardEditors` eventually.  Same with many of the others.

Reply:
i don't know if there already exists a tree editor (or an array editor) example in "StandardEditors" but yeah, these can be moved there.

Reply:
correction - there already exists a `"TreeEditor_demo"` which is more or less a copy of this file - and the differences need to be reconciled and then this file can probably be removed.